newsroom/templates/agenda_item_print.html

```diff
@@ -73,6 +73,7 @@ <h5>{{ gettext('Coverages:') }}</h5>
     <div>Related Events:</div>
     {%- for event in related_events %}
         <div>
+            Url: <a href="{{ url_for_agenda(event) }}">{{ url_for_agenda(event) }}</a>
```

Comment:
`Url` should be all caps, along with all the other templates

Reply:
done !